### ISSUE TITLE
Feedback on new desired handbook structure

### ISSUE THREAD

**Issue by CoachBirgit** [WordPress/contributor-handbook#14]
# Objective

The DEIB working group committed to update and expand the current Contributor Handbook. We need your feedback and suggestions about the desired structure of the overhauled handbook.

## Feedback required

Please leave your feedback below as a comment or in the [Google Doc with commenting permissions](https://docs.google.com/document/d/1KVo2O-vX0SfFxKNeh0lJxwlYdiA_2FSJ5X5s-Uj3kqk/edit?usp=sharing)

## Notes

- [Initial proposal by Josepha ](https://make.wordpress.org/updates/2021/03/16/proposal-a-wordpress-project-contributor-handbook/)
- [Current state of the Contributor Handbook](https://make.wordpress.org/handbook/)

**Comment by patriciabt** [comment]
I would suggest to include chapters such as:

- Presentation of the WordPress project. Same as for the [last Mentorship program videos](https://make.wordpress.org/community/handbook/contributor-day/contributor-working-group/contributor-mentorship-program-for-wordpress/how-and-where-to-start-contributing/#introduction-to-wordpress): Introduction to WordPress and Release process
- Find your interest, with link to the existing [Find Your Team page](https://make.wordpress.org/contribute/)
- How to start contributing, with resources gathered from the last Mentorship Program, [overview and onboarding videos to teams](https://make.wordpress.org/community/handbook/contributor-day/contributor-working-group/contributor-mentorship-program-for-wordpress/how-and-where-to-start-contributing/)
- A chapter "Don't do it alone" with a presentation of the Contributor Mentorship Program, with links to the upcoming or current calls for cohorts, and potential future on-demand mentorship (see/overlaps? the [Contributor working group tracker](https://github.com/WordPress/wp-contributor-working-group-tracker/) )
- Presentation of DEIB, Sustainability and the Five for the Future initiative

And of course what Josepha mentioned in the initial proposal:

- Community Code of Conduct
- Accessibility Policy
- Diversity and Inclusion Policy
- Privacy Policy
- Conflict of Interest Policy
- Code of Ethics

**Comment by angelasjin** [comment]
As we move forward with this, I took a peek at some other handbooks we have across the WordPress ecosystem, and began to wonder the purpose of this particular handbook. My sense is that the people we want to direct to this handbook are: 
- New to WordPress contributors
- People who have questions about "administrative" or cultural norm topics for the overall project

I'm curious what you all think about that :) 

If my sense is on the right track, I think it would change how we structure the handbook and what information we would include. My general recommendation is that we keep this handbook as lean as possible so that it avoids being overwhelming to new contributors, makes it easy to search for information, and is a handy guide to direct folks to various parts of the project.

**Comment by lada7042** [comment]
I agree with @patriciabt and @angelasjin. Let's think about who is the user? First, it's a new contributor. They need to know what kinds of things they can contribute to, so an overview of the WordPress project. Then they need step by step instructions on How to contribute. Where to get support? The onboarding program. 
The other user is a current contributor who needs information for an event or to help a others.
